docs/index.md

```diff
@@ -676,6 +674,20 @@ The process begins with a QR code or `openid://` link. The only differences are:
     ```
 ---
 
+# FAQ
+
+## Which clinical data should be considered in decision-making?
+* The data in Health Cards should focus on communicating "immutable clinical facts".
+* Each use case will define specific data profiles.
+  * For COVID-19 Vaccination Credentials, the [SMART Health Cards: Vaccination IG"](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main) defines requirements
+* When Health Cards are used in decision-making, the verifier is responsible for deciding what rules to apply
+  * decision-making rules may change over time as our understanding of the clinical science improves
```

Comment:
Shall we add:
* decision-making rules may be subject to rules from international, national and local health authorities

Reply:
Yes, adding this now.